plans.go

```diff
@@ -35,6 +35,7 @@ type Plan struct {
 	Limits              PlanLimits `json:"limits"`
 	TokenPerHour        float64    `json:"token_per_hour"`
 	TokenPerMonth       float64    `json:"token_per_month"`
+	Position            float64    `json:"position"`
```

Comment:
I believe that position should be an integer. I'm wrong?

Reply:
yep, it's float value